docs/testing/django-admin-decommission-plan.md

```
@@ -0,0 +1,301 @@
+replaced, the form cannot be deleted.
+
+---
+
+## 1. Coupling inventory — what references the admin forms today
+
+| Consumer | Reference | Decommission action |
+|---|---|---|
+| `CaseViewSet.create()` | `form = CaseAdminForm(data=…, request=request)` `api_views.py:351` | **Replace** with serializer + `Case.validate()` (Task A1) |
+| Django admin `CaseAdmin` | `form = CaseAdminForm`, inlines, `save_related` gate `admin.py:426,702` | **Delete** custom form/inlines → stock `ModelAdmin` or unregister (Task D) |
+| Django admin `DocumentSourceAdmin` | `form = DocumentSourceAdminForm` `admin.py:874` | **Delete** custom form → stock (Task D) |
+| Custom widgets | `ToastUIEditorWidget`, `MultiTimelineField`, `MultiEvidenceField`, `MultiCourtCaseField`, `MultiTextField`, Nepali date | only used by the admin forms | **Delete** with the forms (Task D) |
```

markdownlint-cli2 (0.22.1)

[warning] Table column count
Expected: 3; Actual: 4; Too many cells, extra data will be missing

(MD056, table-column-count)